### ISSUE TITLE
[feature request] Create ppa for nuclear debian packages

### ISSUE THREAD

**Issue by mkg20001** [nukeop/nuclear#153]
A PPA for automatic updates of the nuclear debian/ubuntu package would be really nice to have.

For now I've [added it](https://github.com/mkg20001/deb.mkg20001.io/commit/627580b6156e3edecdb939338f7ffdb4bd646a71) to my personal PPA where it should appear soon

**Comment by nukeop** [comment]
We could add this to the readme if you want to take care of it permanently.

**Comment by mkg20001** [comment]
> if you want to take care of it permanently.
Yes, I'd love to.
I could create a seperate repository as mine also tends to contain lots of
other packages.

nukeop <notifications@github.com> schrieb am So., 20. Jan. 2019, 02:10:

**Comment by nukeop** [comment]
Ok, open a pull request once you got it done on your end then.

**Comment by mkg20001** [comment]
I've created the repo at https://deb.mkg20001.io/nuclear.
If you prefer to have the repo under your own domain you can do so without installing any software (thanks to the power of IPFS) by adding the following records to your DNS.
```
_dnslink.nuclear-ppa.gumblert.tech TXT   "dnslink=/ipns/deb.mkg20001.io/nuclear"
         nuclear-ppa.gumblert.tech CNAME gateway.ipfs.io
```

(The CNAME can point to any valid IPFS gateway like `cloudflare-ipfs.com` - which has HTTPS)

To install the repo simply run:

```bash
curl -s https://deb.mkg20001.io/key.asc | sudo apt-key add -
echo "deb https://deb.mkg20001.io/nuclear nuclear beta" | sudo tee /etc/apt/sources.list.d/mkg.list
```

Note that the `beta` channel points to the current latest release (pre-release and not pre-release)  while `stable` (currently empty) points to the latest release.

If you want any extra files included (such as a `index.html` page) under `deb.mkg20001.io/nuclear` or your custom domain simply open a pr to https://github.com/mkg20001/deb.mkg20001.io

I'll publish the PR with the changes to the readme once you decide the domain for the PPA

**Comment by chepec** [comment]
@mkg20001 Really appreciate the effort in making Nuclear more easily accessible!

I get a segfault when executing Nuclear from this repo (the installation with apt goes without a hitch).
```
$ lsb_release -a
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.6 (stretch)
Release:	9.6
Codename:	stretch
$ npm -v
6.4.1
$ node -v
v10.15.0
$ nuclear 
Segmentation fault
```
Perhaps it's meant only for Ubuntu? In any case, this is a repo of pre-alpha software, so I'm not asking for help. Just thought I should report the issue, and encourage the effort :-)

**Comment by nukeop** [comment]
@mkg20001  Let's use your domain for simplicity, this way you get all the credit for maintaining this, and this isn't a well-known brand anyway.

@chepec Wait for 0.4.4, I need to update Electron to eliminate this bug.

**Comment by mkg20001** [comment]
@nukeop Okay, thanks! :)
Will create a PR soon!

nukeop <notifications@github.com> schrieb am Di., 22. Jan. 2019, 17:49:

**Comment by mkg20001** [comment]
@nukeop How should I best add the PPA to the README? Under community maintained packages as a subsection? Or add it to the community maintained packes tables directly?

**Comment by nukeop** [comment]
I think it'll be best to add it directly to the table.

**Comment by mknepper** [comment]
Should we make a PPA on Launchpad, too?

**Comment by mkg20001** [comment]
I don't think that it's possible to add the deb as-is without rebuilding.
If that's possible then that could be done as well.

M. Knepper <notifications@github.com> schrieb am Do., 24. Jan. 2019, 12:19:

**Comment by FinixFighter** [comment]
> it is a long-standing issue. The problem is that not everyone likes to use the snap packages. For instances I removed them all from my system and I rather like to install a deb package instead.

Exactly. I never used snaps; at most I use flatpak, if I really need it. But deb package is always first choice for me too.

**Comment by nukeop** [comment]
This issue is from an older version of Nuclear that has been replaced with a complete rewrite. None of the old issues apply to the new codebase.

The old code is preserved on the [legacy/electron](https://github.com/nukeop/nuclear/tree/legacy/electron) branch. If you have feedback about the new version, head to [Discussions](https://github.com/nukeop/nuclear/discussions).
